src/shared_settings_keys.hpp

```diff
@@ -19,15 +19,50 @@ const char cd_volume[]= "cd_volume";
 const char mouse_sensetivity[]= "cl_mouse_speed";
 const char fov[]= "cl_fov";
 
+/* essential keys */
 const char key_forward[]= "in_key_forward";
 const char key_backward[]= "in_key_backward";
 const char key_step_left[]= "in_key_left";
 const char key_step_right[]= "in_key_right";
+const char key_speed_up[]= "in_key_speed_up";
+const char key_always_run[]= "in_key_always_run";
+const char key_jump[]= "in_key_jump";
+const char key_fire[]= "in_key_fire";
+const char key_weapon_1[]= "in_key_weapon_1";
+const char key_weapon_2[]= "in_key_weapon_2";
+const char key_weapon_3[]= "in_key_weapon_3";
+const char key_weapon_4[]= "in_key_weapon_4";
+const char key_weapon_5[]= "in_key_weapon_5";
+const char key_weapon_6[]= "in_key_weapon_6";
+const char key_weapon_7[]= "in_key_weapon_7";
+const char key_weapon_8[]= "in_key_weapon_8";
+const char key_weapon_9[]= "in_key_weapon_9";
+const char key_minimap[]= "in_key_minimap";
+const char key_minimap_scale_dec[]= "in_key_minimap_scale_dec";
+const char key_minimap_scale_inc[]= "in_key_minimap_scale_inc";
+const char key_small_hud_off[]= "in_key_small_hud_off";
+const char key_small_hud_on[]= "in_key_small_hud_on";
+const char key_weapon_change[]= "in_key_weapon_change";
+const char key_weapon_next[]= "in_key_weapon_next";
+const char key_weapon_prev[]= "in_key_weapon_prev";
+const char key_menu[]= "in_key_menu";
+const char key_menu_mouse[]= "in_key_menu_mouse";
+const char key_menu_left[]= "in_key_menu_left";
+const char key_menu_right[]= "in_key_menu_right";
+/* addiotional optional keys unnecessary with mouse */
+const char key_inventory_use[]= "in_key_inventory_change";
```

Comment:
Invertory? _Chasm: the Rift_ has an inventory, like _Heretic_/_Hexen_?
Let's remove unnecessary controls.

Reply:
I am generally against adding additional functionality, that doesn't exist in original game. Even minor changes like sprint or zoom can break original game experience.